content/english/post/linn-birmingham.md

```diff
@@ -0,0 +1,34 @@
++++
+showonlyimage = true
+draft = false
+image = "img/linn_birmingham.jpg"
+date = "2020-05-31T00:00:00-00:00"
+title = "Charles Linn and Confederate Soldiers and Sailors Monument—Birmingham, AL, USA"
+photo_credits = "Jay Reeves/Associated Press"
+photo_source_url = ""
```

Comment:
Is photo_source_url intentionally left out?

Reply:
Oop! No. Will fix.